src/libpmem2/pmem2_utils.h

```diff
@@ -26,7 +26,16 @@ pmem2_assert_errno(void)
 	return -errno;
 }
 
+static inline void
+pmem2_err_clear(void)
+{
+	errno = 0;
+	char *errormsg = (char *)out_get_errormsg();
+	strcpy(errormsg, "\0");
+}
+
 #define PMEM2_E_ERRNO (pmem2_assert_errno())
+#define PMEM2_ERR_CLR (pmem2_err_clear())
```

Comment:
make this into a function-like macro (i.e., `PMEM2_ERR_CLR()`).
This should probably be empty in release builds, right?

Reply:
+1

Reply:
Or do we need macro? Can we just use the inline function?